Makefile

```diff
@@ -1,23 +1,20 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright 2022-2023 IBM Corp.
 CC = gcc
-_CFLAGS = -MMD -O2 -std=gnu99 -Wall -Werror
+_CFLAGS = -MMD -O2 -std=gnu99 -Wall # -Werror
 CFLAGS =
 LDFLAGS =
 
 SECVAR_TOOL = $(PWD)/secvarctl-cov
```

Comment:
Fixed in a commit, but eventually I would prefer to use more include Makefiles to share behaviors, rather than chaining calls unless it makes sense to do so. Would be a lot easier to have `check` targets properly depend on builds, and not have inter-Makefile dependencies.